### ISSUE TITLE
how to cite Stan and DOIs

### ISSUE THREAD

**Issue by mitzimorris** [stan-dev/sgb#9]
how to cite Stan? is a FAQ - cf https://discourse.mc-stan.org/t/how-to-cite-stan/28779

- for many journals, citations need DOIs - we should look into getting DOIs for Stan releases from zenodo - https://about.zenodo.org

- the project would benefit from having single citation, because funding agencies want to use simple metrics

- but reproducible science would benefit from citations to the specific version used

we need clear guidelines on how do cite Stan as well as how to record the specifics of the experiment and results.

**Comment by storopoli** [comment]
My opinion is to point towards one unique citation with DOI for the purpose of item 2.

I am in favor of not having a zenodo DOI for each issue, but having a guideline on the citations instructions that "users should strive to make analyzes reproducible by providing a Stan version along with technical details of the computing environment while citing Stan"

**Comment by spinkney** [comment]
Agh, journals and their need for DOIs, how does one cite tweets, discourse posts, etc.? Anyway, I agree with @storopoli about one unique DOI.

**Comment by leofontenelle** [comment]
> * for many journals, citations need DOIs - we should look into getting DOIs for Stan releases from zenodo - https://about.zenodo.org

Aside from a (medical) journal editor. I don't know of any journal that refuses to receive citations without DOIs. However, DOIs do make it much easier for authors and journals to agree something (else than an article or book) belongs in the references list. Moreover, having DOIs might make it easier to have consistent citation practices, as Zotero (and Mendley? EndNote?) can pull metadata from CrossRef and, IIRC, DataCite.

**Comment by bob-carpenter** [comment]
Thanks, @leofontenelle.  

> Zotero (and Mendley? EndNote?) can pull metadata from CrossRef 

That sounds really useful.  Google Scholar's citations are a mess.

> Aside from a (medical) journal editor. I don't know of any journal that refuses to receive citations without DOIs. 

Me, either, but there are a lot of folks in biomedicine using Stan.

**Comment by leofontenelle** [comment]
> > Aside from a (medical) journal editor. I don't know of any journal that refuses to receive citations without DOIs.
> 
> Me, either, but there are a lot of folks in biomedicine using Stan

Sorry for my English. What I meant was, I don't know of any journal requiring DOI for inclusion in the references list. That was an aside from me, a medical journal editor.

**Comment by bob-carpenter** [comment]
Interesting.  I had heard from biomedical researchers that the journals were super picky about what could be included as a reference, but if nobody requires DOIs, then I'm obviously misremembering.  Maybe it was something about PubMed?  But that doesn't seem right, either.

**Comment by leofontenelle** [comment]
Indeed, many are picky, but I don't know of any journal that require DOIs from journal articles to include them in the references list. Its more like (a) some want the DOI to be included in the references if available; and (b) if something is not a journal article, having a DOI makes both authors and journals more comfortable including it in the references. But there are too many journals out there, I work with clinical medicina and public health (not biomedicine) and I'm from a Latin American country, not Western.

*Leonardo Ferreira Fontenelle*
ORCID iD: 0000-0003-4064-433X <https://orcid.org/0000-0003-4064-433X>
    Twitter: @doutorleonardo <https://twitter.com/doutorleonardo>


Em Qui 8 set. 2022, às 17:55, Bob Carpenter escreveu:

**Comment by jear2412** [comment]
I asked @mitzimorris  about this some days ago. She asked me to please follow the guidelines here:  https://mc-stan.org/about/#how-to-cite-stan

Since this issue was opened the documentation website has been improved, so the URL https://mc-stan.org/docs/ should provide a good landing page. 

When googling "how to cite Stan?" one is pointed to the Stan forums, which lead here. I think Mitzi's advice should work for anyone else.  Should this issue be closed?

**Comment by bob-carpenter** [comment]
I'm re-opening this issue.  The doc currently says:

> To cite the interface or post-processing package you used we recommend using the citation conventions for the language (R, Python, Julia, etc.) associated with the interface or package. For example, to cite RStan use the standard conventions for citing R packages.

We don't have that many interfaces.  So I think it'd be nice to just list how to cite them all.  BridgeStan, for example, is multilingual.

Also, I believe some journals only accept citations to things with DOI or that are in journals or cited on PubMed, etc.

The other things people cite is the papers behind the algorithms.  I think that's basically

* top-level Stan paper from me and others in JStatSoft (of course I have a vested interest in that one)
* autodiff in C++ arXiv paper from me and others
* new R-hat paper that explains how we report R-hat and ESS now
* For algorithms, we have a lot of citations
    - for NUTS, there's the Betancourt conceptual HMC paper and the original NUTS paper of Hoffman and Gelman
    - for ADVI, there's Alp Kucukelbir et al.'s paper
    - for Pathfinder, there's Lu Zhang et al.'s paper

I would probably stop short of including the Workflow paper because that's not Stan-specific.
